Tasks/Grunt/grunttask.ts

```diff
@@ -41,11 +47,85 @@ gt.pathArg(gruntFile);
 
 gt.argString(tl.getInput('arguments', false));
 
-gt.exec()
-.then(function(code) {
-	tl.setResult(tl.TaskResult.Succeeded, tl.loc('GruntReturnCode', code));
-})
-.fail(function(err) {
+var enableCoverage = tl.getBoolInput('enableCodeCoverage');
+
+if (enableCoverage) {
+	var testSrc = tl.getPathInput('testFiles', true, false);
+
+	if (os.type().match(/^Win/)) {
+		var istanbul = tl.createToolRunner(tl.which('istanbul', true));
+	} else {
+		var istanbul = tl.createToolRunner(tl.which('./node_modules/istanbul/lib/cli.js', true));
+	}
+
+	istanbul.argString('cover --report cobertura');
+	istanbul.arg('./node_modules/mocha/bin/_mocha');
+	istanbul.arg(testSrc);
+	istanbul.argString('-- --ui bdd -t 5000');
```

Comment:
i think we accept input with custom file pattern. you need to use find functionality to get the files and pass it here?